src/classes/BDI_MappingServiceAdvanced.cls

```diff
@@ -317,6 +235,108 @@ public with sharing class BDI_MappingServiceAdvanced implements BDI_MappingServi
         return fieldMappingInstance.fieldMappingSet;
     }
 
+    private List<Data_Import_Object_Mapping__mdt> getDataImportObjectMappingMetadata (
+        Data_Import_Field_Mapping_Set__mdt diFieldMappingSet) {
+        return [
```

Comment:
Need to align the formatting for these queries as per the apex style guide https://salesforce.quip.com/TkjIACNQIURb#FGVACAY5bs9